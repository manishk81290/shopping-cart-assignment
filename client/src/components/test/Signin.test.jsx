import React from "react";
import Signin from "./../pages/SignIn";
import Adapter from "enzyme-adapter-react-16";
import { shallow, configure } from "enzyme";

configure({ adapter: new Adapter() });

describe("Tests for Signin Component", () => {
  const tree = shallow(<Signin />);
  it("Signin component should match with snapshot", () => {
    expect(tree).toMatchSnapshot();
  });
});
