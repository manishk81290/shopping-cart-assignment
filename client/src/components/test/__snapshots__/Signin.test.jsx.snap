// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for Signin Component Signin component should match with snapshot 1`] = `ShallowWrapper {}`;
